c2corg_ui/templates/helpers/view.html

```diff
@@ -215,8 +188,25 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
         <div class="list-item" ng-repeat="doc in ${associations}" ng-class="{'new-item': doc['new']}">
           <app-card app-card-doc="doc"></app-card>
           % if type == 'articles':
-            <app-delete-association child-doctype="articles" parent-id="${document['document_id']}" child-id="doc.document_id"
-              ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+              <app-delete-association child-doctype="articles" parent-id="${document['document_id']}" child-id="doc.document_id"
+                ng-if="userCtrl.auth.isAuthenticated()
+              % if is_collaborative(document):
+                  && userCtrl.auth.isModerator()
+
+              % elif is_personal(document):
+                  && userCtrl.auth.isModerator()
+                  || userCtrl.hasEditRights
```

Comment:
As you can see, this part is repeated 4 times for different document types, I think I should find a way to write a small function that will output this.